packages/web-components/src/components/va-telephone/va-telephone.tsx

```diff
@@ -0,0 +1,124 @@
+import {
+  Component,
+  Event,
+  EventEmitter,
+  Fragment,
+  Prop,
+  h,
+} from '@stencil/core';
+
+@Component({
+  tag: 'va-telephone',
+  styleUrl: 'va-telephone.css',
+  shadow: true,
+})
+export class VaTelephone {
+  /**
+   * 3 or 10 digit string representing the contact number
+   */
+  @Prop() contact!: string;
+
+  /**
+   * Optional phone number extension
+   */
+  @Prop() extension: number;
+
+  /**
+   * Indicates if the phone number can be clicked or not
+   */
+  @Prop() notClickable: boolean = false;
```

Comment:
There's some interesting discussion around the naming of this prop in [this slack thread](https://dsva.slack.com/archives/C5HP4GN3F/p1638466045185900).